src/pg_to_evalscript/javascript_processes/ndvi.js

```diff
@@ -0,0 +1,99 @@
+function ndvi(arguments) {
+  const { data, nir = "nir", red = "red", target_band = null } = arguments;
+
+  validateParameter({
+    processName: "ndvi",
+    parameterName: "data",
+    value: data,
+    nullable: false,
+    required: true,
+  });
+
+  validateParameter({
+    processName: "ndvi",
+    parameterName: "nir",
+    value: nir,
+    nullable: false,
+    allowedTypes: ["string"],
+  });
+
+  validateParameter({
+    processName: "ndvi",
+    parameterName: "red",
+    value: red,
+    nullable: false,
+    allowedTypes: ["string"],
+  });
+
+  validateParameter({
+    processName: "ndvi",
+    parameterName: "target_band",
+    value: target_band,
+    allowedTypes: ["string"],
+  });
+
+  const nirBand = data.getBand({ commonName: nir });
```

Comment:
Changed the function so it checks for both `name` and `common_name` in the metadata.

Reply:
I don't think this behaviour is what was meant. e.g, let's say we have this bands metadata:
```python
bands_metadata = [
    {name: "B1", common_name: "nir"},
    {name: "B2", common_name: "red"}, 
    {name: "B3", common_name: "B2"}
]
```
1. If user doesn't specify anything, the defaults `nir="nir"` and `red="red"` are used. So we search such `name` s and find nothing, but we find bands `B1` and `B2` based on the `common_name`
2. User can instead specify `nir="B1"` and `red="B2"`. We find a  band with  `name=B1`. However, we find a band with `name=B2` and another one with `common_name=B2`. Because `name` takes preference, we take band `B2` over `B3`. 

So as I said above, imo we should search both by `name` and `common_name`, and in case we get the result for both, we take the one with that `name`, else whichever one we found.

Reply:
I believe that the latest version should handle this workflow.